.circleci/config.yml

```diff
@@ -61,10 +61,15 @@ jobs:
           export CSC_KEY_PASSWORD=$APPLE_CODE_SIGNING_PASSWORD
           npm run build
       - run: |
+          echo "Creating release notes..."
```

Comment:
Can we rename the job to `deploy-mac-and-brew`?

Reply:
@lolski addressed in https://github.com/graknlabs/workbase/pull/399/commits/ba5bfb0b9d1e8abbcdedd2936e8261bea8590a15